Sming/SmingCore/Network/MqttClient.h

```diff
@@ -54,6 +60,7 @@ class MqttClient: protected TcpClient
 	uint8_t *current;
 	int posHeader;
 	MqttStringSubscriptionCallback callback;
+	int keepAlive = 50;
```

Comment:
this is two and a half times the previous value, does it really need to be increased this much? 20 second is quite a while to wait for a modern network, 50s seems excessive